Research/Research/RSDResourceTransformer.swift

```diff
@@ -196,10 +195,10 @@ extension RSDResourceTransformer {
     ///     - data: The returned Data for this resource.
     ///     - resourceType: The resource type.
     /// - throws: `RSDResourceTransformerError` if the file cannot be found.
-    public func resourceData(ofType defaultExtension: String? = nil, bundle: Bundle? = nil) throws -> (data: Data, resourceType: RSDResourceType) {
+    private func _resourceData(ofType defaultExtension: RSDResourceType?, bundle: RSDResourceBundle?) throws -> (data: Data, resourceType: RSDResourceType) {
```

Comment:
If we're making this private, should we also make the above description regular comments rather than doc comments?

Reply:
I like keeping the doc comment style. You don't have to include them in the documentation, but using that format makes it easier to toggle this, or to change the method signature.